daml-lf/transaction-test-lib/src/main/scala/lf/value/test/ValueGenerators.scala

```diff
@@ -106,8 +106,16 @@ object ValueGenerators {
       .chooseNum(Time.Timestamp.MinValue.micros, Time.Timestamp.MaxValue.micros)
       .map(Time.Timestamp.assertFromLong)
 
+  // generate a builtinException with arbitrary value
+  private def builtinExceptionGen(nesting: Int): Gen[ValueBuiltinException[ContractId]] =
```

Comment:
:+1:

Reply:
But did you see that it is not enabled further down the file. It causes test failure
Perhaps you can help me figure out what is going wrong?!

Reply:
I saw, but there is the TODO so I am happy.